sdks/wasm/Mono.WebAssembly.DebuggerProxy/MonoProxy.cs

```diff
@@ -92,6 +92,7 @@ protected override async Task<bool> AcceptEvent (SessionId sessionId, string met
 				}
 
 			case "Debugger.breakpointResolved": {
+					Log ("debug", "BREAKPOINT RESOLVED");
```

Comment:
This can be removed.

Reply:
Oh yea, my changes to Mono Proxy were not supposed to be included in the PR. They were part of an older commit. Nothing is supposed to change.